beetsplug/subsonicupdate.py

```diff
@@ -29,23 +29,32 @@
 
 import requests
 
+from binascii import hexlify
 from beets import config
 from beets.plugins import BeetsPlugin
 
 __author__ = 'https://github.com/maffo999'
+authtokenversion = '1.12'
 
 
 class SubsonicUpdate(BeetsPlugin):
     def __init__(self):
         super(SubsonicUpdate, self).__init__()
-
         # Set default configuration values
         config['subsonic'].add({
             'user': 'admin',
             'pass': 'admin',
             'url': 'http://localhost:4040',
         })
-
+        self.version = self.__get_version()
+        if self.version > authtokenversion:
+            self._log.info(
+                    u'use token authent method')
+            self.auth = "token"
+        else:
+            self.auth = "password"
+            self._log.info(
+                    u'use password authent method')
```

Comment:
Minor rewording and style suggestion:
```suggestion
        if self.version > authtokenversion:
            self.auth = "token"
        else:
            self.auth = "password"
        self._log.info(u"using '{}' authentication method"
                .format(self.auth))
```

Is `> 1.12` the right version check, or should it rather be `>= 1.13`?

Reply:
Should be `<= 1.12`, so `> 1.12` is fine!

Reply:
I'm asking because the API docs state that token authentication is new in `1.13.0`. There doesn't seem to be a version `1.12.1`, so this should indeed work. I don't know subsonic's versioning and release scheme, so no idea whether there's a chance that a `1.12.1` API version will ever be released (probably not, since `1.16.1` is current).